### ISSUE TITLE
Testing on juicefs + es, then es service crashes frequently

### ISSUE THREAD

**Issue by yanming-zhang** [juicedata/juicefs#1837]
The log information is as follows：
![image](https://user-images.githubusercontent.com/11608532/164051620-a1cdce70-984b-4801-9db6-c365acc041f2.png)

**Comment by yanming-zhang** [comment]
After referring to https://www.elastic.co/guide/en/elasticsearch/reference/7.16/index-modules-store.html, by adding the parameter node.store.allow_mmap: false, although the service no longer crashes, the performance will get worse

**Comment by davies** [comment]
What's the version of kernel? Can you try to disable the kernel cache by `--attr-cache=0`

Also, collecting the access log while reproducing the bug will be very helpful.

**Comment by yanming-zhang** [comment]
I have set --attr-cache=0, but still have such problems

**Comment by yanming-zhang** [comment]
my linux kernel version:  3.10.0-862.14.4, may be a bug in the kernel?

**Comment by davies** [comment]
It's possible, can you `cat /jfs/.accesslog` while reproducing the problem? then post the access log here

**Comment by yanming-zhang** [comment]
upgraded kernel version from 3.10.0-862 to 5.4.190,  
jdk version from openjdk-17.0.1 to oraclejdk-17.0.3, 
but still have such problems

**Comment by davies** [comment]
Please collect the accesslog, or tell us the exact steps to reproduce the issue, thanks!

**Comment by davies** [comment]
And what's the mount options?

**Comment by ccl0326** [comment]
```
_netdev,allow_other,cache-dir=/data/jfs-cache,cache-size=921600,attr-cache=0,entry-cache=0,dir-entry-cache=0,get-timeout=120,put-timeout=120,io-retries=60,max-uploads=60,max-deletes=12,buffer-size=16384,upload-limit=2048,prefetch=30,no-usage-report,metrics=0.0.0.0:9567
```

**Comment by fghosth** [comment]
I have the same problem。

**Comment by davies** [comment]
We can confirm that this is a bug, will fix it.

**Comment by davies** [comment]
This should be fixed by #2336
